### ISSUE TITLE
Virtual Hub Capacity

### ISSUE THREAD

**Issue by Akashc0807** [hashicorp/terraform-provider-azurerm#23282]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Description

HI Team,

I am creating a virtual hub, But not able to see Virtual hub capacity in the capacity. 

### New or Affected Resource(s)/Data Source(s)

azurerm_virtual_hub

**Comment by rcskosir** [comment]
Thank you for taking the time to open this issue. We use Issues in this repository to track feature enhancements and bugs in the Azure Provider. So that we can maintain focus on that, we instead ask that broader questions are raised using one of the [Community Resources](https://www.terraform.io/community), such as the [Azure Provider forum](https://discuss.hashicorp.com/c/terraform-providers/tf-azure/34). 
There's not enough information in this issue for me to understand what you are intending. Are you requesting for capacity to be added as a feature enhancement on to the azurerm_virtual_hub?

**Comment by Akashc0807** [comment]
Indeed, As per me. I can't add or modify the virtual hub capacity from terraform. Also, it is rewriting the capacity to 2 after running the terraform file again. 


is this feature enhancement will be available in next version or any timeline ?

**Comment by Akashc0807** [comment]
![MicrosoftTeams-image (5)](https://github.com/hashicorp/terraform-provider-azurerm/assets/114410741/eef47476-9d5a-4a6c-b9ff-207e078dbb49)

**Comment by neil-yechenwei** [comment]
Thanks for raising this issue. Seems it has been supported by [virtual_router_auto_scale_min_capacity ](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_hub#virtual_router_auto_scale_min_capacity)of azurerm_virtual_hub.

**Comment by rcskosir** [comment]
Thank you for taking the time to raise this! I am going to close this with @neil-yechenwei‘s response as an answer. If you have future questions, I suggest using the [Community Resources](https://www.terraform.io/community), such as the [Azure Provider forum](https://discuss.hashicorp.com/c/terraform-providers/tf-azure/34).
